test/cookbooks/test/recipes/php.rb

```diff
@@ -0,0 +1,24 @@
+::Chef::Recipe.include Apache2::Cookbook::Helpers
+
+include_recipe 'php'
```

Comment:
This isn't needed since we're directly installing it, please remove it.

Reply:
With this removed, the php packages get installed after `_install` cleans up the distro directories, so it does it on the 2nd converge and fails the idempotency check. Should I bump the php suite to 3 converges or leave this in?

Reply:
We should probably implement the same workaround we have in our wrapper cookbook in this one since it will fix that problem. i.e.:
```ruby
directory 'purge distro conf.modules.d' do
  path "#{apache_dir}/conf.modules.d"
  recursive true
  action :nothing # trigger this in other recipes as needed
end

directory 'purge distro conf.d' do
  path "#{apache_dir}/conf.d"
  recursive true
  action :nothing # trigger this in other recipes as needed
end
```

Then we can just notify this resource.